cmd/runtimetest/main.go

```diff
@@ -320,6 +322,35 @@ func validateROPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateOOMScoreAdj(spec *rspec.Spec) error {
+	logrus.Debugf("validating oomScoreAdj")
+	if spec.Linux.Resources != nil && spec.Linux.Resources.OOMScoreAdj != nil {
+		expected := *spec.Linux.Resources.OOMScoreAdj
+		f, err := os.Open("/proc/1/oom_score_adj")
+		if err != nil {
+			return err
+		}
+		defer f.Close()
+
+		s := bufio.NewScanner(f)
+		for s.Scan() {
+			if err := s.Err(); err != nil {
+				return err
+			}
+			text := strings.TrimSpace(s.Text())
+			actual, err := strconv.Atoi(text)
+			if err != nil {
+				return err
+			}
+			if actual != expected {
+				return fmt.Errorf("oomScoreAdj expected: %v, actual: %v", expected, actual)
+			}
+		}
+	}
```

Comment:
What happens if `oomScoreAdj` is `null` or unset?  Do we require particular runtime behavior in that case?  I'd expect we require the runtime to leave the setting alone, although the current spec doesn't spell that out.